python/interpret_community/mlflow/mlflow.py

```diff
@@ -0,0 +1,12 @@
+import mlflow
+from tempfile import TemporaryDirectory
+import os
+
+from ..explanation.explanation import save_explanation
+
+
+def _log_explanation(name, explanation):
+    with TemporaryDirectory() as tempdir:
+        path = os.path.join(tempdir, 'explanation')
+        save_explanation(explanation, path)
+        mlflow.log_artifacts(path)
```

Comment:
seems a bit strange to have a log without a download, but if the idea is to do this incrementally I'm fine with that

Reply:
Yeah, get_explanation is a more complicated operation. The idea here is that we're demonstrating that we can easily get files into MLflow - a reasonably comfortable dev should be able to get them back if they really wanted to use this right now. Obviously not everyone wants to go through that process, though, which is why this function is internal for now.